.github/workflows/java-wasm-bindings.yml

```diff
@@ -24,15 +24,15 @@ jobs:
           bundler-cache: true
 
       - name: rake templates
-        run: PRISM_SERIALIZE_ONLY_SEMANTICS_FIELDS=1 bundle exec rake templates
+        run: PRISM_EXCLUDE_PRETTYPRINT=1 PRISM_SERIALIZE_ONLY_SEMANTICS_FIELDS=1 bundle exec rake templates
 
       - name: Set up WASI-SDK
         run: |
           wget https://github.com/WebAssembly/wasi-sdk/releases/download/wasi-sdk-25/wasi-sdk-25.0-x86_64-linux.tar.gz
           tar xvf wasi-sdk-25.0-x86_64-linux.tar.gz
 
       - name: Build the project
-        run: make java-wasm WASI_SDK_PATH=$(pwd)/wasi-sdk-25.0-x86_64-linux PRISM_SERIALIZE_ONLY_SEMANTICS_FIELDS=1
```

Comment:
`PRISM_SERIALIZE_ONLY_SEMANTICS_FIELDS=1` has to be specified somewhere or the resulting AST created will include syntax tool info in addition to what we need for the parser so serialization will not work.  Is this maybe happening elsewhere?

Reply:
Correct, this one was redundant as already applied here:
https://github.com/ruby/prism/pull/3516/files#diff-236daf15d21928259835dcaf94f20bd485d8b3f33ed4df0ccece2278f1d0262bR27